Date: Feb 06 21:53:34 2019

From: 8faf9c60a22b60d511041f338c9ff64fcf137e57 Feb 20 16:34:11 2019

Subject: [PATCH] XONE-19941 : ntpd crash on pthread warmup call

	Disable pthread warmup api as it is causing thread unwinding on pthread_cancel

Source: COMCAST

Upstream-Status: Pending

Signed-off-by: mselva006c <mani_selvaraj@comcast.com>
---

Index: ntp-4.2.8p11/ntpd/ntpd.c
===================================================================
--- ntp-4.2.8p11.orig/ntpd/ntpd.c
+++ ntp-4.2.8p11/ntpd/ntpd.c
@@ -34,9 +34,6 @@
 # ifdef HAVE_PTHREAD_H
 #  include <pthread.h>
 # endif
-# if defined(linux)
-#  define NEED_PTHREAD_WARMUP
-# endif
 #endif
 
 #ifdef HAVE_UNISTD_H
